PhysicsTools/NanoAOD/python/l1trig_cff.py

```diff
@@ -0,0 +1,179 @@
+import FWCore.ParameterSet.Config as cms
+from PhysicsTools.NanoAOD.nano_eras_cff import *
+from PhysicsTools.NanoAOD.common_cff import *
+
+l1_float_precision_=12
+print("l1pre",l1_float_precision_)
```

Comment:
Thanks for the update! Can you just please remove this print?

Reply:
urgh sorry,  of course!